source/partials/_cookie_wrapper.html.erb

```diff
@@ -0,0 +1,11 @@
+<script>
+  document.addEventListener('DOMContentLoaded', function () {
+    <% markup = capture do %>
+      <%= partial 'partials/cookie_banner' %>
+    <% end %>
+    var markup = `<%= markup.squish %>`;
```

Comment:
Note that the backtick doesn't work in IE11, if we're still supporting this browser.

Reply:
@andysellick No worries. I think you may be able to JSON stringify the `markup`, and then parse it back to a regular string when it's inserted into the DOM, but I appreciate IE11 isn't a high priority these days.